WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Color+DesignSystem.swift

```diff
@@ -21,11 +21,12 @@ public extension Color {
                 if AppConfiguration.isJetpack {
                     return jetpack
                 } else {
-                    return jetpack // FIXME: WordPress colors
+                    return wordPress
                 }
             }
 
             private static let jetpack = Color(DesignSystemColorNames.Background.jetpack)
+            private static let wordPress = Color(DesignSystemColorNames.Background.wordPress)
```

Comment:
💡 Instead of having both `jetpack` and `wordpress` constants that are available to both `Jetpack` and `WordPress` builds, we can do the following instead:

1. Remove the `jetpack` and `wordpress` constants.
2. Refactor the `brand` static variable to this:
```swift
static let brand = Color(DesignSystemColorNames.Background.brand)
```
3. Then we can have 2 `Colors.xcassets` files, one for WordPress and another for Jetpack. Those files should have a different `brand` color. We can also have a `SharedColors.xcassets` that contain colors shared between Jetpack and WordPress.

That being said, I realize this is out of scope of this PR and perhaps should be discussed with @alpavanoglu. I recommend to work on this change in a separate PR.

Reply:
That's a good idea!

I agree it's out of scope of this PR, but it's something to consider as we evolve the Design System 👍